chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/cluster_setup

```diff
@@ -98,25 +98,25 @@ EOF
 
 # Copy the repo to the test node (don't want to have to clone every test run - impacts Gerrit)
 # We exclude chroma-externals and copy just what we need to reduce the io load on our builders.
-TEST_DEPS="chroma/chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies"
-(tar -czf - --exclude='chroma/chroma-externals*' .; tar -czf - $(sed -e 's|^|chroma/chroma-externals/|' < $TEST_DEPS); tar -czf - chroma/chroma-externals/*.zip chroma/chroma-externals/*.tar.*) | ssh chromatest@$CHROMA_MANAGER "mkdir -p chroma_test_env; tar -C chroma_test_env -xizf -"
+TEST_DEPS="$REL_CHROMA_DIR/chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies"
```

Comment:
Assuming you found [it](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/18/files/03ce98415b4757c907c315c7cedae6e2da68ceb3#diff-8045fd024ec11b9c80a47e4de53b1945R29) but just want to make sure the question was answered.